### PR TITLE
Stairs: Code cleanup, improve descriptions

### PR DESCRIPTION
**Pull request by paramat**
"Wooden Stair Inner" -> "Inner Wooden Stair"
"Wooden Stair Outer" -> "Outer Wooden Stair"

### CODE REVIEW

File: mods/default/nodes.lua [hunk 1]
```diff
@@ -1902,6 +1902,7 @@ function default.chest.register_chest(name, d)
 			local meta = minetest.get_meta(pos)
 			meta:set_string("infotext", "Locked Chest")
 			meta:set_string("owner", "")
+			meta:mark_as_private("owner")
```
Comment (sfan5) [inline mods/default/nodes.lua]: what's the point if the infotext reveals the owner anyway?

Comment (sfan5) [inline mods/default/nodes.lua]: what's the point if the infotext reveals the owner anyway?

Comment (paramat) [reply mods/default/nodes.lua]: Ok thanks, will edit.

**Comment by paramat** [comment]
Updated but unsure what the argument should be. documented as 'name' but also as 'vars', perhaps 'variable name' so "formspec"?

**Comment by SmallJoker** [comment]
@paramat What formspec meta value? There's none set - neither for locked nor unlocked chests.

**Comment by paramat** [comment]
Yes that was confusing me. Does this need an argument at all? It seems to need a 'var' 'name' to set what 'var' (variable?) is private. Like i said, i don't understand =)

**Comment by paramat** [comment]
Updated, removed meta commit as invalid apparently.

**Comment by paramat** [comment]
0132bfe41e03f400dbef85354f47ea44d962b0ce

**Pull request closed without merging**